src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -369,6 +369,76 @@
     end
   end
 
+  describe 'POST #save_distribution' do
+    let(:action) { post :save_distributions, project: apache_project, distributions: ['openSUSE_Tumbleweed'] }
+
+    before do
+      login admin_user
+      Project.any_instance.stubs(:save_distributions).returns(true)
+      Project.any_instance.stubs(:prepend_kiwi_config).returns()
```

Comment:
What returns this? `nil`? If so, I prefer to see it, then pass `nil` to the `returns` method